Branch-SDK/src/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -71,6 +71,26 @@ public boolean handleBranchViewIfAvailable(ServerResponse resp) {
         return isBranchViewShowing;
     }
 
+    @Override
+    public void onRequestSucceeded(ServerResponse response, Branch branch) {
+        // Check for any Third party SDK for data handling
+        String appOrigin = "";
+        try {
+            if (response.getObject().has(Defines.Jsonkey.AppOrigin.getKey())) {
+                appOrigin = response.getObject().getString(Defines.Jsonkey.AppOrigin.getKey());
+            }
+            boolean isFabricEnabled = appOrigin.equals(Defines.Jsonkey.AppOriginTwitterFabric.getKey());
```

Comment:
I agree.  This was our shared narrative around a true partnership - where things work automagically for our mutual customers.  @aaustin: do you want to jump on a call to talk?